app/_hub/moesif/kong-plugin-moesif/_metadata.yml

```diff
@@ -23,6 +23,9 @@ privacy_policy_url: https://www.moesif.com/privacy?utm_medium=docs&utm_campaign=
 
 terms_of_service_url: https://www.moesif.com/terms?utm_medium=docs&utm_campaign=partners&utm_source=kong
 
+free: true
```

Comment:
Sorry, missed this as well. If you're setting `enterprise: true`, you can't also set `free: true`. The `free: true` setting removes all tier badges.

```suggestion
```

Reply:
@lena-larionova sorry didn't quite follow. What is the right approach for a plugin that supports both OSS and enterprise tiers?

Reply:
We set `free: true` and `enterprise: false`. This is because anything that is available in the free tier is _also_ available in all other tiers. 

The Enterprise badge normally marks plugins that are _only_ available in a paid tier.

Reply:
@lena-larionova just pushed a fix